docs/src/modules/components/ExampleCollection.js

```diff
@@ -167,7 +157,6 @@ export default function ExampleCollection() {
                       href={example.tsLink}
                       variant="body2"
                       sx={{
-                        fontFamily: 'IBM Plex Sans',
                         fontWeight: 500,
```

Comment:
it would be better to use `theme.typography.fontWeightMedium`:

```suggestion
                        fontWeight: 'medium',
```